src/main/java/com/vinylteam/vinyl/web/servlets/ContactUsServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.vinylteam.vinyl.web.servlets;
+
+import com.vinylteam.vinyl.entity.User;
+import com.vinylteam.vinyl.entity.UserPost;
+import com.vinylteam.vinyl.service.UserPostService;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+
+import java.io.IOException;
+
+public class ContactUsServlet extends HttpServlet {
+    private final UserPostService service;
```

Comment:
For future. Please add empty row after class declaration and before closing it. It is kind of our code style